### PR TITLE
[NetManager] Fix Jira support button issue

### PR DESCRIPTION
**Pull request by ja-odur**
#### Summary of Changes (What does this PR do?)
- Add Jira support button on every page
-  Fixes this [issue](https://github.com/airqo-platform/AirQo-frontend/issues/192)

#### Is this change ready to hit production in its current state?
- [ ] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [ ] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [PLAT-417](https://airqoteam.atlassian.net/browse/PLAT-417)

#### Screenshots (optional)

### CODE REVIEW

**Review by danielogen — APPROVED** [review]
Working well as expected. Thank for the fix @ja-odur

File: netmanager/src/utils/customHooks.js [hunk 1]
```diff
@@ -5,3 +5,51 @@ export function useInitScrollTop() {
...
+export function useJiraHelpDesk() {
+  useEffect(() => {
+    // if (user._id != {}) {
+    function jiraHelpdesk(callback) {
+      let jhdScript = document.createElement("script");
+      jhdScript.type = "text/javascript";
+      jhdScript.setAttribute("data-jsd-embedded", null);
+      jhdScript.setAttribute(
+        "data-key",
+        "cf4a44fc-f333-4e48-8e6c-6b94f97cea15"
```
Comment (Baalmart) [inline netmanager/src/utils/customHooks.js:18]: Are we supposed to have this key stored here? environmental variables perhaps?

File: netmanager/src/App.js [hunk 2]
```diff
@@ -74,141 +75,152 @@ if (localStorage.jwtToken) {
...
-              <Route exact path="/" component={Landing} />
-              <Route
+const App = () => {
+
+  useJiraHelpDesk();
+
+  return (
+    <Provider store={store}>
+      <ThemeProvider theme={theme}>
+        <Router>
```
Comment (Baalmart) [inline netmanager/src/App.js:85]: Thanks, I guess we might have to move this router code to the Routers folder in the near future. Just to maintain a short and sweet App.js. :)

**Review by Baalmart — APPROVED** [review]
Looks good to me, I have just left a few comments....that might need to be transferred to issues I guess.

**Review by Baalmart — CHANGES_REQUESTED** [review]
But on second thought, I think we need to first address the security concern before we proceed with integration....

**Comment by ja-odur** [comment]
> @Baalmart Since these keys are used on the altlassian page and they also provided script-tag with the same information to be embedded into html, I dont think it's a security threat but @danielogen can verify this.

@Baalmart Since these keys are used on the altlassian page and they also provided script-tag with the same information to be embedded into html, I dont think it's a security threat but @danielogen can verify this.

**Comment by Baalmart** [comment]
Okay, let us wait for the verification, it will be interesting to learn about keys which are meant to be publicly accessible. Awaiting new thoughts/findings on this.

> @Baalmart Since these keys are used on the altlassian page and they also provided as script-tag with the same information to be embedded into html, I dont think it's a security threat but @danielogen can verify this.

**Comment by ja-odur** [comment]
> Okay, let us wait for the verification, it will be interesting to learn about keys which are meant to be publicly accessible. Awaiting new thoughts/findings on this.
> 
> > @Baalmart Since these keys are used on the altlassian page and they also provided as script-tag with the same information to be embedded into html, I dont think it's a security threat but @danielogen can verify this.

Alright as we wait. Let me hide the keys

**Comment by danielogen** [comment]
> > Okay, let us wait for the verification, it will be interesting to learn about keys which are meant to be publicly accessible. Awaiting new thoughts/findings on this.
> > > @Baalmart Since these keys are used on the altlassian page and they also provided as script-tag with the same information to be embedded into html, I dont think it's a security threat but @danielogen can verify this.
> 
> Alright as we wait. Let me hide the keys

I wasn't able to find any information about the key for now but I would suggest we go-ahead and hide it -  as I can see from some GitHub issues, guys are not revealing it either.

**Comment by ja-odur** [comment]
@Baalmart @danielogen  I have hidden the key but we still have this information in other earlier branches. Not sure how we can resolve that.

**Comment by Baalmart** [comment]
Thanks, I guess we shall rebase the rest of the branches immediately after we get this PR into staging....
> @Baalmart @danielogen I have hidden the key but we still have this information in other earlier branches. Not sure how we can resolve that.

File: netmanager/src/utils/customHooks.js [hunk 3]
```diff
@@ -5,3 +5,48 @@ export function useInitScrollTop() {
...
 }
+
+export function useJiraHelpDesk() {
+  useEffect(() => {
+    // if (user._id != {}) {
+    function jiraHelpdesk(callback) {
+      let jhdScript = document.createElement("script");
+      jhdScript.type = "text/javascript";
+      jhdScript.setAttribute("data-jsd-embedded", null);
+      jhdScript.setAttribute("data-key", process.env.REACT_APP_JIRA_KEY);
```
Comment (danielogen) [inline netmanager/src/utils/customHooks.js:16]: LGTM

**Review by Baalmart — APPROVED** [review]
Looks good, thanks.

**Pull request merged**